frappe/desk/doctype/bulk_update/bulk_update.py

```diff
@@ -22,7 +22,7 @@ def update(doctype, field, value, condition='', limit=500):
 		frappe.throw(_('; not allowed in condition'))
 
 	docnames = frappe.db.sql_list(
-		'''select name from `tab{0}`{1} limit 0, {2}'''.format(doctype, condition, limit)
```

Comment:
Here it is not trivially easy to migrate to qb due to the condition being dynamically supplied. I noticed this is used by client code - is it acceptable to leave this as-is?

Reply:
qb can accept dynamic fields like this `table[fieldname]`

Reply:
Makes sense but the variable in this case is something like "field > 5" - but it looks like Pypika allows this using Criterion.any. Will attempt to convert that way

Reply:
Criterion.any does not appear to work when an entire string is supplied - only when a pypika field is supplied with some constant value. E.g. this is supported Criterion.any(table[fieldname] == 'value') but this is not: Criterion.any(""" 'fieldname' = 'value' """)

Therefore I don't really see a way to refactor this locally. Please advise if qb can be used here.